src/main.rs

```diff
@@ -319,7 +319,7 @@ async fn main() -> Result<()> {
         }
         Some("inspect") => {
             if let Some(matches) = matches.subcommand_matches("inspect") {
-                let uri = matches.value_of("uri").unwrap();
+                let uri = matches.value_of("id").unwrap();
```

Comment:
nitpicking, maybe we should rename the `uri` variable to be `id`, just to be consistent with the cli arg